cmd/crates/soroban-spec-typescript/fixtures/test_custom_types/src/index.ts

```diff
@@ -631,6 +627,22 @@ export interface Client {
 
 }
 export class Client extends ContractClient {
+  static async deploy<T = Client>(
+    /** Constructor/Initialization Args for the contract's `__constructor` method */
+    args: null,
```

Comment:
```suggestion
    args: null,
```
I think for this we could not have the `args` parameter at all and then use `null` in the forwarded call.

So perhaps we should add a second example that has args?

Reply:
Good idea!